pkg/server/routes.go

```diff
@@ -488,9 +488,13 @@ func (s *Server) routes(
 	fileUploadHandler := handlers.NewFileUploadHandler(
 		base,
 		services.NewCreateFileUploadURL(serviceConfig, s3Client),
+		services.NewCreateUploadedFile(serviceConfig, store.CreateUploadedFile),
 	)
 	api.Handle("/file_uploads", fileUploadHandler.Handle())
 
+	api.HandleFunc("/file_uploads/presignedurl", fileUploadHandler.GeneratePresignedURL).
+		Methods("POST")
+
```

Comment:
So due to the way this flow works with AWS we have multiple `POST` needs on this path, which means I needed to break up the generation of a pre-signed URL from the saving of the file metadata on successful upload. 

Spinning up a whole new handler with a `Handle()` doing matching on a single http verb felt too heavy here, so I just broke it out like this. Up for suggestions though.